src/core/server/opensearch/opensearch_config.ts

```diff
@@ -246,6 +250,13 @@ export class OpenSearchConfig {
    */
   public readonly shardTimeout: Duration;
 
+  /**
+   * Set of options to configure memory circuit breaker for query response.
+   * The percentage filed is to determine the threshold for maximum heap size for memory circuit breaker.
```

Comment:
spelling: ~`filed`~ -> `field`

Reply:
Also did you want quotes around `'percentage'` similar to `'enabled'`?

Reply:
will update